python/pyspark/sql/tests/test_functions.py

```diff
@@ -83,7 +83,13 @@ def test_function_parity(self):
         missing_in_py = jvm_fn_set.difference(py_fn_set)
 
         # Functions that we expect to be missing in python until they are added to pyspark
-        expected_missing_in_py = set()
+        expected_missing_in_py = {
+            # TODO: listagg functions will soon be added and removed from this list
```

Comment:
Let's file a JIRA, and make `TODO` as `TODO(SPARK-XXXXX): ...`

Reply:
added https://issues.apache.org/jira/browse/SPARK-50220